lib/autoscale/aws/autoscaler.go

```diff
@@ -203,32 +203,33 @@ func (a *Autoscaler) WaitUntilInstanceTerminated(ctx context.Context, instanceID
 	return trace.Wrap(err)
 }
 
-// GetJoinToken fetches and decrypts cluster join token from SSM parameter
-func (a *Autoscaler) GetJoinToken(ctx context.Context) (string, error) {
-	name := a.tokenParam()
-	a.Debugf("GetJoinToken(%v)", name)
-	resp, err := a.SystemsManager.GetParameterWithContext(ctx, &ssm.GetParameterInput{
-		Name:           aws.String(name),
+// GetServiceURLAndJoinToken fetches and decrypts cluster service URL and join token from SSM parameter
+func (a *Autoscaler) GetServiceURLAndJoinToken(ctx context.Context) (string, string, error) {
```

Comment:
I would probably add here variable names for output parameters, otherwise, it is not clear what is string one and string two.

Reply:
Good point. Fixed.